packages/@aws-cdk/aws-redshift/lib/cluster-ref.ts

```diff
@@ -0,0 +1,52 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as secretsmanager from '@aws-cdk/aws-secretsmanager';
+import { IResource } from '@aws-cdk/core';
+import { Endpoint } from './endpoint';
+
+/**
+ * Create a Redshift Cluster with a given number of nodes.
+ */
+export interface ICluster extends IResource, ec2.IConnectable, secretsmanager.ISecretAttachmentTarget {
+  /**
+   * Name of the cluster
+   * @attribute ClusterName
+   */
+  readonly clusterName: string;
+
+  /**
+   * The endpoint to use for read/write operations
+   * @attribute EndpointAddress,EndpointPort
```

Comment:
`.` instead of `,`?

Reply:
Honestly I don't even know what `@attribute` does. It is just a copypasta from the `@aws-cdk/aws-rds` package:
https://github.com/aws/aws-cdk/blob/c176fdf232cc78680b1df1ed81d838d26f1ec4d9/packages/%40aws-cdk/aws-rds/lib/cluster-ref.ts#L20-L24

I thought it somehow referenced the CloudFormation return values (https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-redshift-cluster.html#aws-resource-redshift-cluster-return-values). 

Since in a lot of packages it is just left blank I would do this now too, i.e.

```suggestion
   * @attribute 
```

Reply:
...well I guess I just found out, bc the JSII compiler had something to say:

```
error: [awslint:resource-attribute:@aws-cdk/aws-redshift.Cluster.clusterEndpointAddress] resources must represent all cloudformation attributes as attribute properties. "@attribute ATTR[,ATTR]" can be used to tag non-standard attribute names
```

So I recon the previous way of doing it, i.e.
```suggestion
   * @attribute EndpointAddress,EndpointPort
```
was correct?

Reply:
That's actually our Construct Library linter, not JSII :). I thought it would be `.`, but if that makes the build fail, of course disregard my comment, and use `,` :)